lib/base-compiler.js

```diff
@@ -291,23 +297,31 @@ Compile.prototype.compile = function (source, options, backendOptions, filters)
                 }
                 return result;
             })
-            .then(_.bind(self.postProcessAsm, self))
+            .then(_.bind(function(result){return self.postProcessAsm(result, filters);}, self))
             .then(function (result) {
                 if (result.okToCache) self.env.cachePut(key, result);
                 return result;
             });
     });
 };
 
-Compile.prototype.postProcessAsm = function (result) {
+Compile.prototype.postProcessAsm = function (result, filters) {
+    var self = this;
     if (!result.okToCache) return result;
-    var demangler = this.compiler.demangler;
+    var demangler = self.compiler.demangler;
     if (!demangler) return result;
-    return this.exec(demangler, [], {input: _.pluck(result.asm, 'text').join("\n")})
+    return self.exec(demangler, [], {input: _.pluck(result.asm, 'text').join("\n")})
         .then(function (demangleResult) {
             var lines = utils.splitLines(demangleResult.stdout);
             for (var i = 0; i < result.asm.length; ++i)
                 result.asm[i].text = lines[i];
+            result.supportCfg = false;
```

Comment:
Minor English point here: ideally the variable would be `supportsCfg` and the function `supportsCFGGeneration`. If it's easy to rename, do so please, else it's fine as it. Also preferably can we keep to the same capitalisation convention: if the variable is `supportsCfg` the function should be `supportsCfgGeneration()`. I tend to use Google's convention of having abbreviations be mixed case and not ALLCAPS, so I'd prefer `supportsCfgGeneration()`.

Reply:
done